inc/apirest.class.php

```diff
@@ -350,7 +350,7 @@ public function parseIncomingParams($skip_check_content_type = false) {
          $content_type = $_SERVER['HTTP_CONTENT_TYPE'];
       } else {
          if (!$skip_check_content_type) {
-            $this->returnError("No Content-Type header found", 400, "ERROR_CONTENT_TYPE_NOT_FOUND");
+            $content_type = "application/json";
```

Comment:
Thanks @orthagh.
I think this is not only creating a security issue ^^
So it's ok.